docs/api/alerting/list_rule_types.asciidoc

```diff
@@ -1,143 +1,174 @@
-
-NOTE: Some rule types are limited to specific features.  These rule types are not available when <<create-edit-rules, defining rules>> in <<management,Stack Management>>.
-
 [[list-rule-types-api-request]]
-==== Request
+=== {api-request-title}
 
 `GET <kibana host>:<port>/api/alerting/rule_types`
 
 `GET <kibana host>:<port>/s/<space_id>/api/alerting/rule_types`
 
+=== {api-prereq-title}
+
+If you have `read` privileges for one or more {kib} features, the API response
+contains information about the appropriate rule types. For example, there are
+rule types associated with the *Management* > *Stack Rules* feature,
+*Analytics* > *Discover* and *{ml-app}* features, *{observability}*, and
+*Security* features. To get rule types associated with the
+*{stack-monitor-app}* feature, use the `monitoring_user` built-in role.
+
+For more details, refer to <<kibana-feature-privileges>>.
+
+=== {api-description-title}
+
+Each rule type includes a list of authorized consumer features. For each feature,
+users are authorized to perform either `read` or `all` operations on rules of
+that type. This enables you to determine which rule types you can read, create,
+or modify.
+
+// TBD: What is the purpose of this note? Since we're not creating/defining rules in this API, can it be removed?
+NOTE: Some rule types are limited to specific features, which might not be
+available when <<create-edit-rules,defining rules>> in *{stack-manage-app}*. 
+
 [[list-rule-types-api-params]]
-==== Path parameters
+=== {api-path-parms-title}
 
 `space_id`::
-  (Optional, string) An identifier for the space. If `space_id` is not provided in the URL, the default space is used.
+(Optional, string) An identifier for the space. If `space_id` is not provided in
+the URL, the default space is used.
+
+[[list-rule-types-response]]
+=== {api-response-body-title}
+
+Each rule type has the following properties in the API response:
+
+`action_groups`::
+(array of objects) An explicit list of groups for which the rule type can
+schedule actions, each with the action group's unique ID and human readable name.
+Rule `actions` validation uses this configuration to ensure that groups are
+valid. Use `kbn-i18n` to translate the names of the action group when
+registering the rule type.
+//TBD: The kbn-i18n phrase doesn't seem appropriate for this context. Can we remove it?
+
+`action_variables`::
+(object) An explicit list of action variables that the rule type makes available
+via context and state in action parameter templates, and a short human readable
+description. The Rule UI uses this information to prompt users for these
+variables in action parameter editors. Use `kbn-i18n` to translate the
+descriptions.
+//TBD: What is the intent of "explicit list"? Can we drop "explicit"?
+//TBD: Which "Rule UI" does this refer to? Is it only the "Rules and Connectors" UI in Stack Management? Or do all apps use this information?
```

Comment:
Rule UI is the Create/Edit Rule flyout.

Reply:
Thanks! I've clarified that text (and dropped the "explicit") in e4fea08e725a63fc88eeb3ffc9108da41ba4be4c